src/ui/templates/components/yes-no-radio-buttons.njk

```diff
@@ -29,9 +31,12 @@
     {% set noValue = 'No' %}
   {% endif %}
 
+  {% if not params.notInlineRadios %}
```

Comment:
Can we call this `horizontalRadios` so it's inline with GOV design descriptions?